src/components/film-screen/film-screen.jsx

```diff
@@ -64,30 +63,13 @@ const FilmScreen = (props) => {
               </p>
 
               <div className="movie-card__buttons">
-                <button
-                  className="btn btn--play movie-card__button"
-                  type="button"
-                  onClick={(evt) => {
-                    evt.preventDefault();
-                    handlePlayBtnClick(id);
-                  }}
-                >
-                  <svg viewBox="0 0 19 19" width="19" height="19">
-                    <use xlinkHref="#play-s"></use>
-                  </svg>
-                  <span>Play</span>
-                </button>
-                <button
-                  className="btn btn--list movie-card__button"
-                  type="button"
-                  onClick={handleMyListBtnClick}
-                >
-                  <svg viewBox="0 0 19 20" width="19" height="20">
-                    <use xlinkHref="#add"></use>
-                  </svg>
-                  <span>My list</span>
-                </button>
-                <Link to={AppRoute.FILMS + id + AppRoute.REVIEW} className="btn movie-card__button">Add review</Link>
+                <PlayButton id={id} onPlayButtonClick={handlePlayButtonClick} />
+                <MyListButton id={id} isFavorite={isFavorite} onFavoritesButtonClick={onButtonClick} />
+                {
+                  authorizationStatus === AuthorizationStatus.AUTH
+                    ? <Link to={`${AppRoute.FILMS}${id}${AppRoute.REVIEW}`} className="btn movie-card__button">Add review</Link>
+                    : ``
```

Comment:
Можно написать и так
```suggestion
                  authorizationStatus === AuthorizationStatus.AUTH
                    && <Link to={`${AppRoute.FILMS}${id}${AppRoute.REVIEW}`} className="btn movie-card__button">Add review</Link>
```